frontend/src/components/ProgressBar/ProgressBar.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const ProgressBar = props => {
+  const { bgcolor, completed, label } = props;
```

Comment:
please, try to destructure it at the argument, instead of props add `({ bgcolor, completed, label }) => `
**and don't forget the icons**